pkgs/build-support/rust/build-rust-crate/helpers.nix

```diff
@@ -3,23 +3,23 @@
   kernel = stdenv.hostPlatform.parsed.kernel.name;
   abi = stdenv.hostPlatform.parsed.abi.name;
   cpu = stdenv.hostPlatform.parsed.cpu.name;
-   updateFeatures = f: up: functions: builtins.deepSeq f (lib.lists.foldl' (features: fun: fun features) (lib.attrsets.recursiveUpdate f up) functions);
+   updateFeatures = f: up: functions: lib.deepSeq f (lib.foldl' (features: fun: fun features) (lib.attrsets.recursiveUpdate f up) functions);
    mapFeatures = features: map (fun: fun { features = features; });
-   mkFeatures = feat: lib.lists.foldl (features: featureName:
+   mkFeatures = feat: lib.foldl (features: featureName:
      if feat.${featureName} or false then
        [ featureName ] ++ features
      else
        features
-   ) [] (builtins.attrNames feat);
-  include = includedFiles: src: builtins.filterSource (path: type:
-     lib.lists.any (f:
+   ) [] (lib.attrNames feat);
+  include = includedFiles: src: lib.filterSource (path: type:
+     lib.any (f:
        let p = toString (src + ("/" + f));
        in
        p == path || (lib.strings.hasPrefix (p + "/") path)
      ) includedFiles
   ) src;
-  exclude = excludedFiles: src: builtins.filterSource (path: type:
-    lib.lists.all (f:
+  exclude = excludedFiles: src: lib.filterSource (path: type:
```

Comment:
`lib.filterSource` is not defined and I'm getting an error for my carnix-generated project.

```
error: attribute 'filterSource' missing, at /nix/store/cyzias1yfbljig7jzi6iagfqwqimw16g-unstable-20.03pre208171.2e8fc97dbfa/unstable/pkgs/build-support/rust/build-rust-crate/helpers.nix:21:33
```

```nix
# Generated by carnix 0.10.0: carnix generate-nix --src ./.
# ...
buildRustCrate {
   # ...
   src = exclude [ ".git" "target" ] ./.;
   # ...
}
# ...
```

Reply:
oh wow, sorry. I thought I fixed that since I ran into the exact same issue. I'll push a fix later.